tests/ratingservice/main_test.py

```diff
@@ -19,52 +19,60 @@
 import requests
 import json
 
-class TestEndpoints(unittest.TestCase):    
+
+class TestEndpoints(unittest.TestCase):
     def testGetRating(self):
         """ Test if getting the rating of a product returns success """
         products = read_products()
         for product in products:
-            url = "https://ratingservice-dot-{0}.wl.r.appspot.com/rating/{1}".format(getProjectId(), product)
+            url = "https://ratingservice-dot-{0}.wl.r.appspot.com/rating/{1}".format(
+                getProjectId(), product)
             res = requests.get(url)
             self.assertEqual(res.status_code, 200)
 
     def testGetRatingNotExist(self):
         """ Test if getting non-existing product returns 404 """
-        url = "https://ratingservice-dot-{0}.wl.r.appspot.com/rating/{1}".format(getProjectId(), "random")
+        url = "https://ratingservice-dot-{0}.wl.r.appspot.com/rating/{1}".format(
+            getProjectId(), "random")
         res = requests.get(url)
         self.assertEqual(res.status_code, 404)
-    
+
     def testRate(self):
         """ Test if rating a product returns success """
         products = read_products()
         for product in products:
-            url = "https://ratingservice-dot-{}.wl.r.appspot.com/rating".format(getProjectId())
+            url = "https://ratingservice-dot-{}.wl.r.appspot.com/rating".format(
+                getProjectId())
             res = requests.post(url, data={
-                'score' : 5,
-                'id' : product
+                'rating': 5,
+                'id': product
             })
             self.assertEqual(res.status_code, 200)
-    
-    def testGetRatingCorrect(self):
-        """ Test if getting the rating of a product returns a correct number """
-        products = read_products()
-        url_get = "https://ratingservice-dot-{0}.wl.r.appspot.com/rating/{1}".format(getProjectId(), products[0])
-        url_post = "https://ratingservice-dot-{0}.wl.r.appspot.com/rating".format(getProjectId())
-        res1 = requests.get(url_get).json()
-        requests.post(url_post, data={
-            'score' : 5,
-            'id' : products[0]
-        })
-        res2 = requests.get(url_get).json()
-        # The original total score (count * rating) plus the current score 5 must equal to the current total score
-        self.assertTrue(abs(float(res1['rating']) * float(res1['count']) + 5 
-                        - float(res2['rating']) * float(res2['count'])) < 1e-5)
-    
+
+    # def testGetRatingCorrect(self):
+    #     """ Test if getting the rating of a product returns a correct number """
```

Comment:
are you planning to resurrect this test after it's adapted to new code?

Reply:
yes. i replaced the constant values (rating 5 with 1 vote) by randomized rating numbers. after reviewing the goal of the test, and introducing required changes i will return it. this is one of the reasons why i drafted the PR.